### ISSUE TITLE
Update link test as per new test flow

### ISSUE THREAD

**Issue by amitkrout** [openshift/odo#1700]
[kind/Enhancement]

## Which functionality do you think we should update/improve?
link test

## Why is this needed? 
To achieve more odo test coverage

**Comment by amitkrout** [comment]
blocked on #1634 and #1698

**Comment by girishramnani** [comment]
closing this as its duplicate of #1616

**Comment by kadel** [comment]
:-D Sorry @amitkrout, wanted to close it almost at the same time, ended up reopening it 😇

**Comment by amitkrout** [comment]
np :)
